### ISSUE TITLE
I am unconvinced that this branding effort does what it sets out to do

### ISSUE THREAD

**Issue by bruceravel** [NSLS-II/artwork#4]
Here's what the branded wallpaper looks like at 6BM with a typical arrangement of windows and things.

![branding](https://user-images.githubusercontent.com/638186/96592423-5edfb880-12b6-11eb-8a0f-df53ff5cac7b.png)

In what sense is this an effective branding exercise?  The floor tab is covered.  Most of the text is covered.

**Comment by stuwilkins** [comment]
@bruceravel  I agree! I think we need to work with Cara to get these changed. She said that we can get to this with the designers in November. 

For now, I think @stuartcampbell and I were thinking that we get the technical machinery in place then a change becomes easy as it's just a case of rebuilding the RPM and then do a mass rollout

I think we need to have the branding on the bottom which is what I think you are saying!
